mathics/builtin/no_meaning.py

```diff
@@ -1,42 +1,61 @@
 # -*- coding: utf-8 -*-
-# Note: this will be redone soon to pull no-meaning operator information from operators.yml out
-# of the MathicsScanner project.
 """
 Operators without Built-in Meanings
 
 Not all operators recognized by the Mathics3 are associated with functions that have built‐in meanings.
-You can use these operators as a way to build up your own notation within the Mathics3.
+You can use these operators as a way to build up your own notation within Mathics3.
 """
 
-from mathics.core.attributes import A_NO_ATTRIBUTES
-from mathics.core.builtin import InfixOperator
-
-
-class Star(InfixOperator):
-    r"""
-    Star <url>
-    :WML link:
-    https://reference.wolfram.com/language/ref/Star.html</url>
-
-    <dl>
-      <dt>'Star[$x$, $y$, ...]'
-      <dd>displays $x$ ⋆ $y$ ⋆ ...
-    </dl>
-
-    >> Star[x, y, z]
-     = x ⋆ y ⋆ z
-
-    >> a \[Star] b
-     = a ⋆ b
-    """
-
-    attributes = A_NO_ATTRIBUTES
-    default_formats = False  # Don't use any default format rules. Instead, see below.
-    formats = {
-        (("InputForm", "OutputForm", "StandardForm"), "Star[args__]"): (
-            'Infix[{args}, "⋆"]'
-        ),
-    }
-
-    operator = "⋆"  # \u22C6
-    summary_text = "star symbol"
+# WARNING: This module uses fancy MetaProgramming to create the Mathics3 Builtin classes for the
+# operators in that have no pre-defined meaning. This is tricky and probably fragile code.
+# We use the type() function to create the class, and setattr, to include this class inside
+# this module.
+
+from sys import modules
+
+import mathics.core.parser.operators
+from mathics.core.builtin import (
+    NoMeaningInfixOperator,
+    NoMeaningPostfixOperator,
+    NoMeaningPrefixOperator,
+)
+from mathics.core.parser.operators import OPERATOR_DATA
+
+# Generate no-meaning Mathics3 Builtin class from the operator name,
+# affix, and Operator Unicode values found in OPERATOR_DATA.  This
+# data ultimately comes from a YAML file in the MathicsScanner project
+# which is processed into a JSON file.
+
+for affix, format_fn, operator_base_class in (
```

Comment:
This loop should be inside a `def init()` function, called at the end of the module.
Otherwise, the global variable `generated_operator_class` is seen by `admin-tools/build_and_check_manifest.py` a duplication of one of the classes defined here (`Square`)
and raise an error.

After this change, just run `admin-tools/build_and_check_manifest.py` and the SYMBOL_MANIFEST.txt must be automatically updated.